java/ql/test/experimental/query-tests/security/CWE-522-DecompressionBombs/pom.xml

```diff
@@ -0,0 +1,79 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
You don't need to add a `pom.xml` file. If what you want is that the dependencies are resolved, you need to create stubs instead. See https://github.com/github/codeql/blob/main/java/ql/src/utils/stub-generator/makeStubs.py for doing that.